pkg/extensions/main.go

```diff
@@ -25,6 +25,7 @@ import (
 	_ "github.com/russellhaering/goxmldsig"
 	_ "github.com/stretchr/testify/require"
 	_ "github.com/vectordotdev/go-datemath"
+	_ "github.com/weaveworks/common/user"
```

Comment:
_(Needed to use some bits from that package, like [`InjectIntoGRPCRequest`](https://github.com/weaveworks/common/blob/master/user/grpc.go#L25) or [`InjectOrgID`](https://github.com/weaveworks/common/blob/7c2720a9024d7fdf5f4668321a4fcddf7b461b27/user/id.go#L38))_

It's not strictly necessary to get the other bits on this pull request working, but quite related: if there would be a gRPC client abstraction/implementation here, it would be necessary, but that's on the Ent repo. 

So, I preferred to avoid having to create a separate pull request just to add that dependency, since it doesn't add lot of complexity to review it and it's all related with the same issue/feature request.

That being said, I'd be happy with only copying the needed bits wherever are needed, if that's what you prefer.

cc/ @sakjur 

Thanks! 🙏🏻